### PR TITLE
WT-12836 Enable prefetch by default in wt verify

### PR DESCRIPTION
**Pull request by etienneptl**

### CODE REVIEW

File: src/utilities/util_main.c [hunk 1]
```diff
@@ -84,7 +84,8 @@ main(int argc, char *argv[])
     size_t len;
     int ch, major_v, minor_v, tret, (*func)(WT_SESSION *, int, char *[]);
     char *p, *secretkey;
-    const char *cmd_config, *config, *p1, *p2, *p3, *readonly_config, *rec_config, *salvage_config;
+    const char *conn_config, *cmd_config, *p1, *p2, *p3, *readonly_config, *rec_config,
```
Comment (kommiharibabu) [inline src/utilities/util_main.c:87]: `cmd_config` must be the first parameter.

File: src/utilities/util_main.c [hunk 2]
```diff
@@ -106,7 +107,7 @@ main(int argc, char *argv[])
         return (EXIT_FAILURE);
     }
 
-    cmd_config = config = readonly_config = salvage_config = secretkey = NULL;
+    conn_config = cmd_config = readonly_config = salvage_config = session_config = secretkey = NULL;
```
Comment (kommiharibabu) [inline src/utilities/util_main.c:110]: same here.

**Review by kommiharibabu — APPROVED** [review]
LGTM

File: src/utilities/util_main.c (hunk 1 above)
Comment (etienneptl) [reply src/utilities/util_main.c]: 9cd762e94f4e505925286ddde5a806638b8c5359

File: src/utilities/util_main.c (hunk 2 above)
Comment (etienneptl) [reply src/utilities/util_main.c]: 9cd762e94f4e505925286ddde5a806638b8c5359

**Review by mm-ng — APPROVED** [review]
LGTM

**Review by clarissecheah — APPROVED** [review]
LGTM! Thanks Etienne

**Pull request merged**